app/controllers/events_controller.rb

```diff
@@ -108,6 +109,26 @@ def send_rejection_emails
     render :email
   end
 
+  # POST /events/1/upload_material
+  def upload_material
+    event = Event.find(params[:event_id])
+    material_path = event.material_path
+    Dir.mkdir(material_path) unless File.exists?(material_path)
+
+    file = params[:file_upload]
+    unless is_file?(file)
+      redirect_to event_path(event), alert: t("events.material_area.no_file_given")
```

Comment:
Outside from views it is always a good idea to use the `I18n` namespace for `t()` methods.